### PR TITLE
Updating directions for new app registration blade

### PR DESCRIPTION
**Pull request by AllisonAm**
@jmprieur Please review when you get the chance. Thank you! 

VSTS item: https://identitydivision.visualstudio.com/DevEx/_workitems/edit/665388

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,23 +64,36 @@ If you want to use this automation, read the instructions in [App Creation Scrip
...
-
-   - Type a key description (of instance `app secret`),
+1. In the  **Azure Active Directory** pane, click on **[App registrations](https://go.microsoft.com/fwlink/?linkid=2083908)** and choose **New registration**.
+1. When the **Register an application page** appears, enter your application's registration information:
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `TaskTrackerWebApp-GroupClaims`.
+   - Leave **Supported account types** on the default setting of **Accounts in this organizational directory only**.
+   - For the **Redirect URI (optional)**, select Web and enter the base URL for the sample. By default, this sample uses `https://localhost:44322/`.
+1. Click **Register** to create the application.
+1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+1. Select the **Expose an API** section, and:
```
Comment (jmprieur) [inline README.md]: @AllisonAm : I don't think that this sample exposes an API?
it's a Web app which calls Microsoft Graph?

File: README.md [hunk 2]
```diff
@@ -64,23 +64,36 @@ If you want to use this automation, read the instructions in [App Creation Scrip
...
-     so record it as soon as it is visible from the Azure portal.
-1. Configure Permissions for your application. To that extent, in the Settings menu, choose the 'Required permissions' section and then,
-   click on **Add**, then **Select an API**, and type `Microsoft Graph` in the textbox. Then, click on  **Select Permissions** and select **Directory.Read.All**.
+   - When you press the **Add** button, the key value will be displayed, copy, and save the value in a safe location.
+   - You'll need this key later to configure the project in Visual Studio. This key value will not be displayed again, nor retrievable by any other means, so record it as soon as it is visible from the Azure portal.
+1. Configure Permissions for your application. Select the **API permissions** section
+   - Click the **Add a permission** button and then,
+   - Ensure that the **Microsoft APIs** tab is selected
+   - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**
+   - In the **Delegated permissions** section, ensure that the right permissions are checked: **Directory.Read.All**, **Mail.Send**. Use the search box if necessary.
```
Comment (jmprieur) [inline README.md]: @AllisonAm : does the sample require Mail.Send? I have not seen it?

File: README.md (hunk 1 above)
Comment (AllisonAm) [reply README.md]: You are correct. You have to go to Expose an API to change the app ID URI as requested in the original version. I then got carried away based on the template, my apologies.

Question around this step- it was in the original guide, but if that URI is not being used, is there a reason we explicitly set it?

File: README.md (hunk 2 above)
Comment (AllisonAm) [reply README.md]: Updated. Sorry, came from the template!

File: README.md (hunk 1 above)
Comment (jmprieur) [reply README.md]: No you are right, in the new app model, we should not even set it (it's a Web API thing)

**Pull request merged**